packaging/package.sh

```diff
@@ -162,11 +162,12 @@ function packaging() {
     cp "$PWD/$DISTRO/cloud.spec" "$RPMDIR/SPECS"
 
     (cd "$RPMDIR"; rpmbuild --define "_topdir ${RPMDIR}" "${DEFVER}" "${DEFFULLVER}" "${DEFREL}" ${DEFPRE+"$DEFPRE"} ${DEFOSSNOSS+"$DEFOSSNOSS"} ${DEFSIM+"$DEFSIM"} -bb SPECS/cloud.spec)
-    (cd $PWD/../; git reset --hard)
     if [ $? -ne 0 ]; then
+        (cd $PWD/../; git reset --hard)
         echo "RPM Build Failed "
         exit 3
     else
+        (cd $PWD/../; git reset --hard)
```

Comment:
Should the `git reset --hard` be part of the packaging script at all?

Reply:
Technically yes, because if `--use-timestamp` is used actual POM versions are changed and to leave the working directory in pristine state it's needed.

Reply:
I would prefer to remove it, especially if someone is working on their feature/bugfix branch it risks removing changes (and losing code).

Reply:
That's actually a good point, I'll add an extra if condition around it later.